contracts/src/v0.8/tests/VRFLoadTestOwnerlessConsumer.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../VRFConsumerBase.sol";
+import "../interfaces/ERC677ReceiverInterface.sol";
+
+/**
+ * @title The VRFLoadTestOwnerlessConsumer contract.
+ * @notice Allows making many VRF V1 randomness requests in a single transaction for load testing.
+ */
+contract VRFLoadTestOwnerlessConsumer is VRFConsumerBase, ERC677ReceiverInterface {
+  // The price of each VRF request.
```

Comment:
Would be helpful for newcomers to also state that this price is in Juels, the lowest denomination of LINK: `// The price of each VRF request in Juels, the lowest denomination of LINK. 1 LINK = 1e18 Juels.` or something similar.